modules/prismaBidAdapter.js

```diff
@@ -0,0 +1,205 @@
+        account: adunitValue.params.account,
+        tagId: adunitValue.params.tagId,
+        videoExt: adunitValue.params.videoExt,
+        label: adunitValue.adUnitCode,
+        bidId: adunitValue.bidId,
+        auctionId: adunitValue.auctionId,
+        transactionId: adunitValue.transactionId,
+        mediatypes: adunitValue.mediaTypes,
+        bidfloor: adunitValue.params.bidfloor || 0,
+        bidfloorCurrency: adunitValue.params.bidfloorCurrency || 'USD',
+        keywords: adunitValue.params.keywords ? transformBidderParamKeywords(adunitValue.params.keywords) : [],
+      }
+      adUnits.push(foo);
+      if (adunitValue.userIdAsEids) userEids = adunitValue.userIdAsEids;
+    });
+    const payload = {
+      adUnits,
+      // TODO: does the fallback make sense here?
+      href: encodeURIComponent(bidderRequest.refererInfo.page || bidderRequest.refererInfo.topmostLocation)
+    };
+    if (bidderRequest) { // modules informations (gdpr, ccpa, schain, userId)
+      if (bidderRequest.gdprConsent) {
+        payload.gdpr = bidderRequest.gdprConsent.gdprApplies ? 1 : 0;
+        payload.gdprConsent = bidderRequest.gdprConsent.consentString;
+      } else {
+        payload.gdpr = 0;
+        payload.gdprConsent = '';
+      }
+      if (bidderRequest.uspConsent) { payload.uspConsent = bidderRequest.uspConsent; }
+      if (bidderRequest.schain) { payload.schain = bidderRequest.schain; }
+      if (userEids !== null) payload.userEids = userEids;
+    };
+    payload.connectionType = getConnectionType();
+
+    if (test) payload.test = 1;
+    const payloadString = JSON.stringify(payload);
+    return {
+      method: 'POST',
+      url: BIDDER_URL,
+      data: payloadString,
+    };
+  },
+  /**
+         * Unpack the response from the server into a list of bids.
+         *
+         * @param {ServerResponse} serverResponse A successful response from the server.
+         * @return {Bid[]} An array of bids which were nested inside the server.
+         */
+  interpretResponse: function(serverResponse, bidRequest) {
+    const serverBody = serverResponse.body;
+    const bidResponses = [];
+    let bidResponse = null;
+    let value = null;
+    if (serverBody.hasOwnProperty('responses')) {
+      Object.keys(serverBody['responses']).forEach(key => {
+        value = serverBody['responses'][key];
+        const url = `${CACHE_URL}?uuid=${value['uuid']}`;
+        bidResponse = {
+          requestId: value['bidId'],
+          cpm: value['cpm'],
+          currency: value['currency'],
+          width: value['width'],
+          height: value['height'],
+          ttl: value['ttl'],
+          creativeId: value['creativeId'],
+          netRevenue: true,
+          prisma: {
+            'ssp': value['bidder'],
+            'consent': value['consent'],
+            'tagId': value['tagId']
+          },
+          /*
+          meta: {
+            'advertiserDomains': value['adomain']
+          }
+          */
```

Comment:
Could this be included in the PR, defaulting to an empty array if `adomain` does not exist on the response

Reply:
Fixed - https://github.com/prebid/Prebid.js/pull/8681/commits/c002549513498e5610be69a98363efd653932366